package.json

```diff
@@ -1,10 +1,13 @@
 {
   "dependencies": {
-    "cheerio": "^1.0.0-rc.2"
+    "@types/minimatch": "^3.0.3",
+    "cheerio": "^1.0.0-rc.2",
+    "minimatch": "^3.0.4",
```

Comment:
to make importing this internally easier please make sure to set the versions of each of these libs to exactly what we already have internally (otherwise know that an import is going to be more painful than needed)